src/pages/api/sanity/lessons/create.ts

```diff
@@ -174,6 +177,16 @@ async function formatSanityMutationForLessons(
           _type: 'reference',
           _ref: videoId,
         },
+        softwareLibraries: [
+          {
+            _type: 'versioned-software-library',
+            _key: topicKey,
+            library: {
+              _type: 'reference',
+              _ref: topicIds[0],
```

Comment:
Grabs the first topicId in the array since we are considering this the "primary tag"